voting/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name = "vote-circuit"
+name = "voting"
```

```suggestion
name = "voting-circuit"
```

I think it's fine to keep the circuit prefix since that's all it is currently. The folder can still be named `voting` though.